app/templates/base.html

```diff
@@ -20,6 +20,12 @@
   {{ govukHeader({
     'homepageUrl': url_for('main.upload'),
     'serviceUrl': url_for('main.upload'),
```

Comment:
Can we use the index for these instead and let them redirect? (for when we add more pages in future)